app/controllers/v0/institutions_controller.rb

```diff
@@ -4,11 +4,16 @@ module V0
   # rubocop:disable Metrics/ClassLength
   class InstitutionsController < ApiController
     # GET /v0/institutions/autocomplete?term=harv
+
+    def approved_institutions
```

Comment:
This should be a `private` method

Reply:
Anything that appears below `private` is a private method, so adding this here makes all of our public actions private, which we don't want.  You need to remove the `private` declaration above and just move the whole `approved_institutions` method below the `private` declaration lower in the class.